VideoLocker/src/main/java/org/edx/mobile/module/analytics/ISegment.java

```diff
@@ -8,6 +8,8 @@
 
 import org.edx.mobile.util.images.ShareUtils;
 
+import java.util.Map;
+
 public interface ISegment {
```

Comment:
There's no point int renaming it while the interface includes segment-specific classes.